pkg/kubectl/cmd/create_pdb.go

```diff
@@ -60,7 +60,8 @@ func NewCmdCreatePodDisruptionBudget(f cmdutil.Factory, cmdOut io.Writer) *cobra
 	cmdutil.AddValidateFlags(cmd)
 	cmdutil.AddPrinterFlags(cmd)
 	cmdutil.AddGeneratorFlags(cmd, cmdutil.PodDisruptionBudgetV1GeneratorName)
-	cmd.Flags().String("min-available", "1", i18n.T("The minimum number or percentage of available pods this budget requires."))
+	cmd.Flags().String("min-available", "", i18n.T("The minimum number or percentage of available pods this budget requires."))
```

Comment:
Done, added a new generator, and switched to it, but it preserves backward compatibility for now. Will add the deprecation notice under release notes; it also prints out the deprecation notice when invoked without min-available/max-unavailable specified.